src/test/java/shop/RealItemTest.java

```diff
@@ -0,0 +1,30 @@
+package shop;
+
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@Tag("RealItemTest")
+@DisplayName("RealItem Class Tests")
+class RealItemTest {
+    RealItem realItem;
+
+    @BeforeEach
+    public void setup() {
+        realItem = new RealItem();
+    }
+
+    @AfterEach
+    public void cleanUp() {
+        realItem = null;
+    }
+
+    @DisplayName("Test Real Item Weight")
+    @Test
+    public void testRealItem() {
+        double expectedWeight = 9.99;
+        realItem.setWeight(expectedWeight);
+        // checking that the expected weight equals the actual weight that is returned by weight getter
+        assertEquals(expectedWeight, realItem.getWeight());
```

Comment:
testing getters and setters is not a good idea. There is no business logic there. On the other hand, there are not so many methods for test in Real- VirtualItem classes. Try to test custom toString methods for the content of a specific value in a string. Ex.:
`assertTrue(realTestItem.toString().contains("Weight: " + EXPECTED_WEIGHT));`